src/tests/snapshots/all__github_secret_scanning__github_secret_alert_revokes_token-2.snap

```diff
@@ -0,0 +1,23 @@
+---
+source: src/tests/github_secret_scanning.rs
+expression: app.emails_snapshot()
+---
+To: something@example.com
+From: noreply@crates.io
+Subject: Exposed API token found
+Content-Type: text/plain; charset=utf-8
+Content-Transfer-Encoding: 7bit
+
+GitHub has notified us that your crates.io API token bar
+
+has been exposed publicly. We have revoked this token as a precaution.
+
+Please review your account at https://crates.io to confirm that no
+
+unexpected changes have been made to your settings or crates.
+
+
+
+Source type: some_source
+
+URL where the token was found: some_url
```

Comment:
This change also brought to light that our secret scanning emails aren't quite formatted as intended 😅